docs/modules/getting_started/pages/first_steps.adoc

```diff
@@ -0,0 +1,181 @@
+----
+
+Since we are accessing the cluster externally, and NiFi is a bit tricky in terms of allowing access from certain urls, a standard port-forward and access via localhost does not work. The following snippet retrieves the external node ip of one of the NiFi pods and the HTTPS port exposed in the NodePort service:
+
+[source]
+----
+include::example$code/test-nifi.sh[tag=get-nifi-host]
+----
+
+With that stored in another variable, we can access the NiFi REST API in order to retrieve a JWT, which is also stored in a variable for later access:
+
+[source]
+----
+include::example$code/test-nifi.sh[tag=get-nifi-token]
+----
+
+With the JWT ready, the NiFi REST API can be accessed properly. This is done in a loop because the NiFi cluster requires some time in order get fully ready. Leader election and synchronization may take some time. It basically checks that two nodes are successfully registered and connected:
+
+[source]
+----
+include::example$code/test-nifi.sh[tag=nifi-test-loop]
+----
+
+This script will loop some time until the NiFi cluster is fully ready. For completeness, here is the full bash script. Ignore the `#tag::` and `#end::` comments, they are used as templates for this getting started guide:
+
+[source]
+----
+include::example$code/test-nifi.sh[]
+----
+
+Run this script via:
+
+[source]
+----
+include::example$code/getting-started.sh[tag=nifi-tests]
+----
+
+Congratulations, you successfully created and tested your first NiFi cluster.
+
+=== Access NiFi the web interface
+
+You can the connection settings for the NiFi cluster via script or stackablectl.
+
+==== Via kubectl and jq
+
+In order to access the NiFi web interface you can use the following script to extract the `<ip>:<port>` settings.
+
+[source]
+----
+kubectl get svc simple-nifi -o json | jq -r --argfile endpoints <(kubectl get endpoints simple-nifi -o json) --argfile nodes <(kubectl get nodes -o json) '($nodes.items[] | select(.metadata.name == $endpoints.subsets[].addresses[].nodeName) | .status.addresses | map(select(.type == "ExternalIP" or .type == "InternalIP")) | min_by(.type) | .address | tostring) + ":" + (.spec.ports[] | select(.name == "https") | .nodePort | tostring)'
+----
+
+which should return something like this. Just pick one of the `<ip>:<port>` settings representing one NiFi node:
+
+[source]
+----
+172.18.0.2:32595
+172.18.0.5:32595
+----
+
+==== stackablectl
+
+Alternatively, when using stackablectl you can use the service command to get a list of all available endpoints:
+
+[source]
+----
+stackablectl svc list
+----
+
+with should return something like this:
+
+----
+ PRODUCT    NAME         NAMESPACE  ENDPOINTS                       EXTRA INFOS
+
+ nifi       simple-nifi  default    https https://172.18.0.5:32595
+
+ zookeeper  simple-zk    default    zk    172.18.0.5:31377
+----
+
+Then connect to `https://<ip>:<port>/nifi` and you should the NiFi web interface.
+
+image::nifi-web-ui.png[]
```

Comment:
The image is scaled to a width of 738 px. Ideally, the NiFi window is scaled to the exact same size before taking the screenshot. Otherwise the content is hard to see.

Reply:
Created smaller screenshot for better visibility